cli/src/acton.act

```diff
@@ -120,6 +121,11 @@ def write_buildzig(file_cap, build_config: BuildConfig):
         build_zig_zon_template = file.ReadFile(
             file.ReadFileCap(file_cap),
             file.join_path([base_path(file_cap), "builder", "build.zig.zon"])).read().decode()
+
+        # Replace SYSPATH with the relative path to the Acton system installation (Zig requires relative paths)
```

Comment:
We have the gen_buildzigzon function that does a bit of templating, so I'd rather keep it all in there rather than spread it out like this.